app/frontend/schools/settings/components/SchoolCustomize__LinksEditor.res

```diff
@@ -329,11 +329,9 @@ let make = (~kind, ~customizations, ~addLinkCB, ~moveLinkCB, ~removeLinkCB, ~upd
                 placeholder={t("title_placeholder")}
                 onChange={handleTitleChange(send)}
                 value=state.title
-                maxLength=24
-              />
-              <School__InputGroupError
-                message={t("cant_empty_message")} active=state.titleInvalid
+                maxLength=30
```

Comment:
This should have been 24 right 🤔

Reply:
Actually, I changed it to 30 because so that the error message gets shown when user enters greater than `24` characters. I think that is what we are doing at other places. 

If we change it to `24` then the user cannot enter more than 24 characters and, as I am checking for 24 `!StringUtils.lengthBetween(~allowBlank=false, title, 1, 24)` the error message is not shown

what do you think?